docs/modules/ROOT/pages/servlet/oauth2/resource-server/jwt.adoc

```diff
@@ -974,6 +974,7 @@ Java::
 public JwtAuthenticationConverter jwtAuthenticationConverter() {
     JwtGrantedAuthoritiesConverter grantedAuthoritiesConverter = new JwtGrantedAuthoritiesConverter();
     grantedAuthoritiesConverter.setAuthoritiesClaimName("authorities");
+    grantedAuthoritiesConverter.setAuthoritiesClaimDelimiter(",");
```

Comment:
While I agree that sometimes claims can be comma-delimited, that is not the use case that is set up by the description above. Unless otherwise specified, the documentation tries to stay with [the spec](https://datatracker.ietf.org/doc/html/rfc6749#section-3.3) which states that `scope` is space-delimited.

As such, please leave these code snippet changes out for now.

Reply:
Thanks for your review